js/ui/html_editor/formats/image.js

```diff
@@ -1,15 +1,41 @@
 import { getQuill } from "../quill_importer";
+import { isObject } from "../../../core/utils/type";
 
 const quill = getQuill();
 const Image = quill.import("formats/image");
 
 class ExtImage extends Image {
+    static create(data) {
+        const SRC = data && data.src || data;
+        let node = super.create(SRC);
+
+        if(isObject(data)) {
+            if(data.alt) {
+                node.setAttribute("alt", data.alt);
+            }
+
+            if(data.width) {
+                node.setAttribute("width", `${data.width}px`);
+            }
+
+            if(data.height) {
+                node.setAttribute("height", `${data.height}px`);
+            }
+        }
+
+        return node;
+    }
+
     static formats(domNode) {
         let formats = super.formats(domNode);
+        const src = domNode.getAttribute("src");
 
         if(domNode.style["float"]) {
             formats["float"] = domNode.style["float"];
         }
+        if(src) {
```

Comment:
Please put down the string with const `src = domNode.getAttribute("src");` before ` if(src) {`